tests/integration/test_tls.py

```diff
@@ -34,6 +34,7 @@ async def test_deploy_ssl_quorum(ops_test: OpsTest):
             num_units=1,
             config={"ca-common-name": "zookeeper"},
             series=TLS_OPERATOR_SERIES,
+            revision=163,
```

Comment:
**todo**: Could you please add a FIXME comment so that we can easily go back to this line once TLS is fixed?

Reply:
Added.